packages/scripts-mobile/_templates/component/new/test.ejs.t

```diff
@@ -1,5 +1,6 @@
 ---
 to: src/components/<%= name %>/<%= name %>.test.tsx
+sh: npx eslint -c ./.eslintrc.js ./src/components/<%= name %> --ext .ts,.tsx --fix --quiet
```

Comment:
@roblafeve Does this line run eslint automatically after a new component is generated?

Reply:
@blaketarter yes it does. 👍